content/sensu-go/6.0/release-notes.md

```diff
@@ -66,6 +67,62 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.0.0 release notes
+
+**July 28, 2020** &mdash; The latest release of Sensu Go, version 6.0.0, is now available for download.
+
+**PLACEHOLDER FOR RELEASE SYNOPSIS**
+With Sensu Go 6, you can control everything through the API. You can still use configuration management tools to bootstrap agent entities, but you don’t need to! Our new agent entity management feature via the backend configuration API nearly eliminates the need for external (or out-of-band) configuration management for Sensu, empowering you to manage agent entity subscriptions and automate the discovery of system facts without updating agent local configuration files. Run a sensuctl command, click a button in the web UI, or execute a custom check plugin.
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.0.0.
+
+**BREAKING CHANGES FOR SENSU 6.0:**
+
+- The database schema for entities has changed.
+As a result, after upgrading to Sensu 6.0, you will not be able to use your database with older versions of Sensu.
+- In [binary-only distributions][164], the web UI is now a standalone product that is no longer distributed inside the `sensu-backend` binary.
+See the [Sensu Go Web repository][163] for more information.
+- After initial creation, you cannot change your `sensu-agent` entity configuration by modifying the agent's configuration file.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][162]) Sensu now logs a warning when secrets cannot be sent to an agent because mTLS is not enabled.
+- ([Commercial feature][162]) Added [JavaScript functions][169] `sensu.EventStatus`, `sensu.FetchEvent`, and `sensu.ListEvents` to the filter execution environment so you can now query the Sensu event store for other events within the filter namespace.
+- ([Commercial feature][162]) Docker images now set their own default values for environment variables `SENSU_AGENT_API_URL`, `SENSU_BACKEND_API_URL`, `SENSU_BACKEND_ETCD_INITIAL_CLUSTER`, `SENSU_BACKEND_ETCD_ADVERTISE_CLUSTER`, `SENSU_BACKEND_ETCD_INITIAL_ADVERTISE_PEER_URLS`, `SENSU_BACKEND_ETCD_LISTEN_CLIENT_URLS`, and `ETCD_LISTEN_PEER_URLS`.
```

Comment:
```suggestion
- ([Commercial feature][162]) Docker-only Sensu now binds to the hostname of containers instead of `localhost`. Docker images now set their own default values for environment variables `SENSU_AGENT_API_URL`, `SENSU_BACKEND_API_URL`, `SENSU_BACKEND_ETCD_INITIAL_CLUSTER`, `SENSU_BACKEND_ETCD_ADVERTISE_CLUSTER`, `SENSU_BACKEND_ETCD_INITIAL_ADVERTISE_PEER_URLS`, `SENSU_BACKEND_ETCD_LISTEN_CLIENT_URLS`, and `ETCD_LISTEN_PEER_URLS`.
```